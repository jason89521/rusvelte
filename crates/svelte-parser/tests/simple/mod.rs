use svelte_parser::Parser;
#[test]
fn test() {
    insta::glob!("inputs/*.svelte", |path| {
        let source = std::fs::read_to_string(path).unwrap();
        let allocator = oxc_allocator::Allocator::default();
        let mut parser = Parser::new(&source, &allocator);
        let result = parser.parse();
        assert!(result.is_ok());
        let root = result.unwrap();
        insta::assert_json_snapshot!(root)
    });
}
