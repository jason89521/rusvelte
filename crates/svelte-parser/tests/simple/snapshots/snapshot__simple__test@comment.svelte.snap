---
source: crates/svelte-parser/tests/simple/mod.rs
expression: root
input_file: crates/svelte-parser/tests/simple/inputs/comment.svelte
---
{
  "type": "Root",
  "start": 0,
  "end": 18,
  "fragment": {
    "type": "Fragment",
    "nodes": [
      {
        "type": "Comment",
        "start": 0,
        "end": 7,
        "data": ""
      },
      {
        "type": "Comment",
        "start": 7,
        "end": 18,
        "data": "data"
      }
    ]
  }
}
